plugins/nextcloud-rule-exclusions-before.conf

```diff
@@ -1998,7 +2009,8 @@ SecRule REQUEST_FILENAME "@rx /ocs/v[0-9\.]+\.php/references/resolve$" \
     ctl:ruleRemoveTargetById=932200;ARGS:reference,\
     ctl:ruleRemoveTargetById=942430;ARGS:reference,\
     ctl:ruleRemoveTargetById=942431;ARGS:reference,\
-    ctl:ruleRemoveTargetById=942432;ARGS:reference"
+    ctl:ruleRemoveTargetById=942432;ARGS:reference,\
+    ctl:ruleRemoveTargetById=921151;ARGS_GET:reference"
```

Comment:
Why `ARGS_GET` here?

Reply:
The rule checks for `ARGS_GET` and not `ARGS` https://github.com/coreruleset/coreruleset/blob/d33dad89052ecc531641b047efcaae339c254362/rules/REQUEST-921-PROTOCOL-ATTACK.conf#L321